browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -2556,7 +2560,185 @@ TEST_F(KeyringServiceUnitTest, ImportFilecoinAccounts) {
   EXPECT_EQ(default_keyring->GetImportedAccountsNumber(), 0u);
   EXPECT_EQ(service.GetHDKeyringById(brave_wallet::mojom::kFilecoinKeyringId)
                 ->GetImportedAccountsNumber(),
-            3u);
+            amount - 1);
+
+  const base::Value* imported_accounts_value =
```

Comment:
Don't add back the test cases that I removed in https://github.com/brave/brave-core/commit/7db10e913cce47efa1140a75b6d4c2b8cf552ff4